doc/discourse.md

```diff
@@ -6,3 +6,18 @@ That subdomain is hosted by [Flying Circus](https://flyingcircus.io/)
 
 To request permissions or changes to Discourse, talk to one of the [administrators](https://discourse.nixos.org/about).
 
+The administration is handled by 2 Moderators, 1-2 representatives from the Steering Committee and the following Administrators from Flying Circus (Discourse hosting provider) and the Infra Team.
```

Comment:
That Flying Circus is the hoster is already mentioned above the fold.

Reply:
```suggestion
The administration is handled by 2 Moderators, 1-2 representatives from the Steering Committee, and the following Administrators from the sponsoring hosting provider (Flying Circus as mentioned above) and the Infra Team.
```
@jtojnar this is the direction I sort of felt like going in.